core/src/test/java/cz/o2/proxima/transform/UserGatewayIndex.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2017-2021 O2 Czech Republic, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.o2.proxima.transform;
+
+import cz.o2.proxima.repository.EntityAwareAttributeDescriptor.Wildcard;
+import cz.o2.proxima.repository.EntityDescriptor;
+import cz.o2.proxima.repository.Repository;
+import cz.o2.proxima.storage.StreamElement;
+import java.util.Map;
+import java.util.UUID;
+
+public class UserGatewayIndex implements ElementWiseTransformation {
```

Comment:
I know it's just example but can you please add short description of this transform? It  will be very helpful for others.